addons/account/models/account_journal_dashboard.py

```diff
@@ -87,7 +98,9 @@ def get_line_graph_datas(self):
                 short_name = format_date(show_date, 'd MMM', locale=locale)
                 data.append({'x': short_name, 'y':last_balance, 'name': name})
 
-        return [{'values': data, 'area': True}]
+        [graph_title, graph_key] = self._graph_title_and_key()
+        color = '#875A7B' if '+e' in version else '#7c7bad'
```

Comment:
ugly hack to get the primary color in enterprise/community.

I had asked qsm how it should be fetched and he said there was no clean way. That there were plenty of places where it's hard-coded.